internal/service/appfabric/generate.go

```diff
@@ -3,7 +3,6 @@
 
 //go:generate go run ../../generate/tags/main.go -ServiceTagsSlice -AWSSDKVersion=2 -ListTags -UpdateTags
 //go:generate go run ../../generate/servicepackage/main.go
-//go:generate go run ../../generate/tagstests/main.go
```

Comment:
Looks like this may have inadvertently been added as part of: https://github.com/hashicorp/terraform-provider-aws/pull/37666. Let me know if this needs to be part of this PR